$NetBSD: patch-CBLAS_CMakeLists.txt,v 1.1 2021/11/23 22:45:01 wiz Exp $

Without this, math/cblas fails configuring with:
CMake Error: INSTALL(EXPORT) given unknown export "cblas-targets"                                                                                     

--- CBLAS/CMakeLists.txt.orig	2021-06-28 16:39:12.000000000 +0000
+++ CBLAS/CMakeLists.txt
@@ -66,7 +66,7 @@ install(FILES
   COMPONENT Development
   )
 
-install(EXPORT ${CBLASLIB}-targets
-  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${CBLASLIB}-${LAPACK_VERSION}
-  COMPONENT Development
-  )
+#install(EXPORT ${CBLASLIB}-targets
+#  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${CBLASLIB}-${LAPACK_VERSION}
+#  COMPONENT Development
+#  )
